extension/data_loader/mman.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This file ensures that mman.h compatible functions are defined in the global
+// namespace for windows and posix environments.
+
+#pragma once
+
+#include <executorch/runtime/platform/compiler.h>
+
+#ifndef _WIN32
+
+#include <sys/mman.h>
+#include <unistd.h>
+
+ET_INLINE long get_os_page_size(){return sysconf(_SC_PAGESIZE)}
```

Comment:
Now that we're wrapping this, I'd like to return `size_t` instead of `long` to be more consistent with the types that ET uses.